circle.yml

```diff
@@ -0,0 +1,69 @@
+machine:
+  node:
+    version: 4
+
+dependencies:
+  pre:
+    - git clone --depth 1 https://github.com/mozilla/fxa-js-client
+    - cd fxa-js-client && npm run setup
+
+    - git clone --depth 1 https://github.com/mozilla/fxa-content-server
+    - cp fxa-content-server/server/config/local.json-dist fxa-content-server/server/config/local.json
+    - cd fxa-content-server && npm i
+    - cd fxa-content-server && node_modules/.bin/bower i
+    - cd fxa-content-server && CONFIG_FILES=server/config/local.json,server/config/production.json node_modules/.bin/grunt build
+
+    - git clone --depth 1 https://github.com/mozilla/fxa-oauth-server
+    - cd fxa-oauth-server && npm i --production
+
+    - git clone --depth 1 https://github.com/mozilla/fxa-profile-server
+    - sudo apt-get install graphicsmagick
+    - cd fxa-profile-server && npm i --production
+
+    - git clone --depth 1 --branch http https://github.com/vladikoff/browserid-verifier
+    - cd browserid-verifier && npm i --production
+    - cd browserid-verifier && npm i vladikoff/browserid-local-verify#http
+
+    - pip install mozdownload mozinstall
+    - mozdownload --version 50.1.0 --destination firefox.tar.bz2
+    - mozinstall firefox.tar.bz2
+
+    - ulimit -S -n 2048
+
+    - sudo apt-get install expect tightvncserver
+    - mkdir -p $HOME/.vnc
+    - bash fxa-content-server/tests/ci/setvncpass.sh
+    - tightvncserver :1
+    - export DISPLAY=:1
+
+    - npm i -g retry-cli
+
+  override:
+    - npm i
+
+  post:
+    - SIGNIN_UNBLOCK_ALLOWED_EMAILS="^block.*@restmail\\.net$" SIGNIN_UNBLOCK_FORCED_EMAILS="^block.*@restmail\\.net$" npm start 2>&1 | tee $HOME/fxa-auth-server.log:
```

Comment:
Tried without these, but both the js-client and the content-server tests failed.